certified-connectors/QPP NG Integration Hub/ReadMe.md

```diff
@@ -1,57 +1,56 @@
 # Quark Publishing Platform NextGen
-The Quark Publishing Platform NextGen connector offers a comprehensive suite of actions for seamless integration to the Quark Publishing Platform NextGen (QPP NG). Effortlessly retrieve and manage user metadata, asset information, group memberships, attribute values, and more. Streamline your processes by leveraging a wide range of actions, from checking group memberships to uploading and downloading files. Simplify your QPP NG interactions and enhance your workflows with this versatile and powerful connector.
+The Quark Publishing Platform NextGen connector offers a comprehensive suite of actions for seamless integration to the Quark Publishing Platform NextGen (QPP NG). With the help of QPP-NG connector, you can.
```

Comment:
Please fix all the typos and modify readme.md file as per the template in the below link. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Fixed typos, rest the format was already reffered from this sample template.
thanks.